python/tank/errors.py

```diff
@@ -37,3 +37,18 @@ def __init__(self):
         """
         super(TankErrorProjectIsSetup, self).__init__("You are trying to set up a project which has already been set up. "
                                                       "If you want to do this, make sure to set the force parameter.")
+
+class TankAuthenticationError(TankError):
+    """
+    Exception that indicates an error has occurred.
+    """
+    pass
+
+
+class TankAuthenticationDisabled(TankError):
+    """
+    Exception that indicates that interactive authentication has been disabled.
+    """
+    def __init__(self):
+        TankError.__init__(self, "Authentication has been disabled.")
```

Comment:
Hmm, is this right? I grepped the code and the only place this is raised is inside the shotgun_auth module. So it should probably be renamed and moved.